mllib/src/main/scala/org/apache/spark/ml/feature/ChiSqSelector.scala

```diff
@@ -245,14 +266,16 @@ object ChiSqSelector extends DefaultParamsReadable[ChiSqSelector] {
 @Since("1.6.0")
 final class ChiSqSelectorModel private[ml] (
     @Since("1.6.0") override val uid: String,
-    private val chiSqSelector: feature.ChiSqSelectorModel)
+    @Since("3.1.0")val selectedFeatures: Array[Int])
   extends Model[ChiSqSelectorModel] with ChiSqSelectorParams with MLWritable {
 
   import ChiSqSelectorModel._
 
-  /** list of indices to select (filter). */
-  @Since("1.6.0")
-  val selectedFeatures: Array[Int] = chiSqSelector.selectedFeatures
+  var prev = -1
```

Comment:
Doesn't matter, but you can write `require(selectedFeatures.sliding(2).forall(l => l(0) < l(1)), ...)`